### PR TITLE
[www] UI api/v3

### LINKED ISSUE #331: Area chart to display Reward & Accumulated Reward
Make a chart below the search tree to display reward behavior.

### PR DESCRIPTION
**Pull request by sahirgomez1**
### UI - Bump to API/v3
This PR 

- fixes #331, 
- refactors frontend to implement flask API V3. 
- migrates from react-bootstrap V1 to latest react-bootstrap V2.
- implements exporting feature in charts.
- supports updating actions space on active node. 
- Make search tree show only active nodes on each step.

TO DO:
- Start a new session using url parameters to land on a given path.

https://user-images.githubusercontent.com/84393628/126715678-95b591b5-c89a-44ff-b980-51a82941e1d5.mov

**Last**

https://user-images.githubusercontent.com/84393628/127159308-66bbc6b5-9ac0-4271-8a9c-9ddb6ce13cdd.mov

### CODE REVIEW

**Comment by ChrisCummins** [comment]
Hey @sahirgomez1 , is this still WIP or ready for review?

**Comment by sahirgomez1** [comment]
> Hey @sahirgomez1 , is this still WIP or ready for review?

@ChrisCummins It was ready for review, but after a conversation with Somya, some updates will be performed today, we can hold the review until the next meeting.

File: www/frontends/compiler_gym/src/components/Navbars/RewardsNavbar.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react";
```
Comment (ChrisCummins) [inline www/frontends/compiler_gym/src/components/Navbars/RewardsNavbar.js]: MIT license header needed on this and www/frontends/compiler_gym/src/components/Sections/RewardHistoryChart.js

**Review by ChrisCummins — APPROVED** [review]
LGTM, thanks @sahirgomez1! Just a couple of the new files that need license headers, then please merge when happy.

Cheers,
Chris

**Pull request merged**